### PR TITLE
Remove legacy documentation, cleanup

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/administrator/administrate.rst [hunk 1]
```diff
@@ -131,11 +131,8 @@ The layers in the admin interface have the following attributes:
...
   the given layer, used to zoom to visible scale, with WMS if it is empty
   we get the values from the capabilities.
-* ``Disclaimer``: optional, copyright of the layer, used by
-  `Disclaimer <http://docs.camptocamp.net/cgxp/2.2/lib/plugins/Disclaimer.html>`_.
-* ``Identifier attribute field``: field used to identify a feature from the
-  layer, e.g.: 'name', used by
-  `FeaturesWindow <http://docs.camptocamp.net/cgxp/2.2/lib/plugins/FeaturesWindow.html>`_.
+* ``Disclaimer``: optional, copyright of the layer.
+* ``Identifier attribute field``: field used to identify a feature from the layer, e.g.: 'name'.
 * ``Restrictions area``: the areas throw witch the user can see the layer.
```
Comment (jwkaltz) [inline doc/administrator/administrate.rst]: throw witch -> through which

(yes I know it is not part of the PR changes, but I am seeing it now :) )

File: doc/administrator/administrate.rst [hunk 2]
```diff
@@ -131,11 +131,8 @@ The layers in the admin interface have the following attributes:
...
 
 On ``internal WMS`` layer we have the following specific attributes:
```
Comment (jwkaltz) [inline doc/administrator/administrate.rst]: layer -> layers

File: doc/integrator/backend_qgis.rst [hunk 3]
```diff
@@ -88,13 +48,15 @@ Finally, your QGIS project layers CRS should all be in the same CSR. This is sub
 change.
 
 Connect to Postgres database
-*****************************
+****************************
 
 This section is subject to change when the QGIS plugin is available.
 
 The way you should connect QGIS to the database is based on an external file
 called. pg_service.conf located in the home directory. The content of this file
```
Comment (jwkaltz) [inline doc/integrator/backend_qgis.rst]: called. pg_service.conf -> called pg_service.conf

File: doc/integrator/backend_qgis.rst [hunk 4]
```diff
@@ -88,13 +48,15 @@ Finally, your QGIS project layers CRS should all be in the same CSR. This is sub
...
-is as follow::
+is as follow:
```
Comment (jwkaltz) [inline doc/integrator/backend_qgis.rst]: is as follows:

File: doc/integrator/backend_qgis.rst [hunk 5]
```diff
@@ -117,19 +79,16 @@ You can have several sections. A section start with a name with [] and
 finish with a blanck line. This file should be a unix file.
```
Comment (jwkaltz) [inline doc/integrator/backend_qgis.rst]: blanck -> blank

File: doc/integrator/backend_qgis.rst [hunk 6]
```diff
@@ -117,19 +79,16 @@ You can have several sections. A section start with a name with [] and
...
 
 On QGIS desktop, when creating a new PostGIS connection, give it a name and use
-the service name (`geomapfish` in our example) in the connection parameters
+the service name (``geomapfish`` in our example) in the connection parameters
 form.
 
 Copy past this file in the server, change the parameters to fit with the server
```
Comment (jwkaltz) [inline doc/integrator/backend_qgis.rst]: Copy past -> Copy-paste

File: doc/integrator/caching.rst [hunk 7]
```diff
@@ -9,11 +9,11 @@ Headers
 By default most of the elements will be cached for 10 days by the browser.
 
 To change this value for all the application change in the
-``vars_<package>.yaml`` file the ``vars/default_max_age`` value. ``0`` will mean ``no-cache``.
+``vars.yaml`` file the ``vars/default_max_age`` value. ``0`` will mean ``no-cache``.
 The specified value is in seconds.
 
 To change this value for a specific service add the following stricture in the
```
Comment (jwkaltz) [inline doc/integrator/caching.rst]: stricture -> structure

File: doc/integrator/docker.rst [hunk 8]
```diff
@@ -51,7 +51,7 @@ A ``docker-compose.yml.mako`` file is created as a starting point.
 
 If you want to host the database on your local machine, you must add a
 ``dbhost`` and ``dbhost_slave`` entry pointing to ``172.17.0.1`` (your host address for Docker
-container) in your ``vars_<package>.yaml`` file. Then you need to make sure
+container) in your ``vars.yaml`` file. Then you need to make sure
 Postgres is configured to listen on that interface and accepts authentication.
 
 If you want to use an external serveur for the database, just put it is address
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: serveur -> server
it is address -> its address

File: doc/integrator/print.rst [hunk 9]
```diff
@@ -97,6 +69,7 @@ generated by the templating. Then we should do:
    This system works for print v2 but must be adapted for
```
Comment (jwkaltz) [inline doc/integrator/print.rst]: Should we remove this part (is print v2 considered deprecated?)

File: doc/builddoc.rst [hunk 10]
```diff
@@ -1,28 +0,0 @@
-.. _build_doc:
```
Comment (jwkaltz) [inline doc/builddoc.rst]: Are we removing this from our documentation? Or moving it elsewhere?

Comment (sbrunner) [reply doc/builddoc.rst]: The documentation is build by Travis and has this own make rule, then for me it don't needs a specific (and probably unmaintained) documentation...

Comment (jwkaltz) [reply doc/builddoc.rst]: That's fine for me, but I think then you need to remove references to "build_doc", too

Comment (sbrunner) [reply doc/builddoc.rst]: For me it's done, do you see an other one?

**Pull request merged**